### DISCUSSION TITLE
Telegram (or another) channel

### DISCUSSION THREAD

**Discussion by BobbyESP** [JunkFood02/Seal#406] [Ideas]
What about creating a Telegram channel for talking about the app or related things...?

**Comment by Marco-9456** [comment]
> What about creating a Telegram channel for talking about the app or related things...?

I think a Discord server would be better since you have the ability to create more than one channel in it every channel can have a different topic and you can even connect (somehow) your gitlog as you can see in this Screenshot 
![Screenshot_20221210_133351_Discord](https://user-images.githubusercontent.com/41828058/206855432-8c760f39-d8f2-4358-940b-8534b78ed911.png)

**Reply by BobbyESP** [reply]
Yess, I know. It would be a nice idea!
